pkg/models/dashboards.go

```diff
@@ -70,6 +71,7 @@ var (
 	ErrDashboardWithSameNameAsFolder = DashboardErr{
 		Reason:     "Dashboard name cannot be the same as folder",
 		StatusCode: 400,
+		Status:     "name-match",
```

Comment:
this does not seem to ever trigger

Reply:
FYI @ifrost if you want to take a look tomorrow, this seems to trigger only when NOT saving a dashboard in the "General" folder, for each other folder it seems to trigger correctly

Reply:
For the record: the backend team is having a discussion about it what should actually change.